providers/os/resources/os.lr

```diff
@@ -929,6 +931,28 @@ iptables.entry {
   chain string
 }
 
+fstab @defaults("path") {
+  init(path? string)
+  path string
+
+  entries() []fstab.entry
+}
+
+private fstab.entry @defaults("device mountpoint") {
+  // Device referenced in the fstab, e.g. LABEL=rootfs
+  device string
+  // Mountpoint, e.g. '/'
+  mountpoint string
+  // Filesystem type, e.g. ext4
+  fstype string
+  // Mount options, e.g. defaults
```

Comment:
```suggestion
  // Mount options, e.g., defaults
```

Reply:
Could we list all the options? Or point our users somewhere?

Reply:
[The list is quite big](https://man.archlinux.org/man/ext4.5#Mount_options_for_ext4)
Also it's bit tough with the references, since it's linux, there are just too many of them.

Reply:
No problem, we just link to that list!

Reply:
Per our discussion, closing. 😄